demos/theme/index.html

```diff
@@ -88,32 +88,32 @@ <h2 class="mdc-typography--headline demo-component-section__heading">
 
           <div>
             <h3 class="mdc-typography--body1">Default (primary theme color)</h3>
-            <button class="mdc-button">
+            <button class="mdc-button demo-button">
```

Comment:
It is unnecessary to add `demo-*` as a namespace to any of these elements. We never require them for selector specificity in the `css`.

To clarify, just because we have `mdc-*`, doesn't mean we _always_ need `demo-*`. There are of course cases where we need a class specifically for the demo to select on, (`secondary-text-button` for instance), but we do not _also_ need `demo-button` on it as well.